crates/symbolicator-service/tests/integration/snapshots/integration__sourcemap__e2e_node_debugid.snap

```diff
@@ -0,0 +1,21 @@
+---
+source: crates/symbolicator-service/tests/integration/sourcemap.rs
+assertion_line: 433
+expression: response.unwrap()
+---
+stacktraces:
+  - frames:
+      - function: Object.<anonymous>
+        abs_path: /Users/lucaforstner/code/github/getsentry/sentry-javascript-bundler-plugins/packages/playground/öut path/rollup/entrypoint1.js
+        lineno: 73
+        colno: 36
+raw_stacktraces:
+  - frames:
+      - function: Object.<anonymous>
+        abs_path: /Users/lucaforstner/code/github/getsentry/sentry-javascript-bundler-plugins/packages/playground/öut path/rollup/entrypoint1.js
+        lineno: 73
+        colno: 36
+errors:
+  - abs_path: /Users/lucaforstner/code/github/getsentry/sentry-javascript-bundler-plugins/packages/playground/öut path/rollup/entrypoint1.js
+    type: invalid_abs_path
```

Comment:
Thats a good example that we indeed need a better fix for https://github.com/getsentry/symbolicator/issues/1151